src/Microsoft.Extensions.EnvironmentAbstractions/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Abstractions for making code that uses file system and environment testable.",
```

Comment:
Ok, Micorosft.Dotnet.Internal.TestingAbstractions and make everything internal and visible only to tests?